silverback/runner.py

```diff
@@ -106,6 +122,36 @@ async def run(self):
             :class:`~silverback.exceptions.NoTasksAvailableError`:
                 If there are no configured tasks to execute.
         """
+        # Initialize broker (run worker startup events)
+        await self.app.broker.startup()
+
+        # Obtain system configuration for worker
+        result = await run_taskiq_task_wait_result(
+            self._create_system_task_kicker(TaskType.SYSTEM_CONFIG)
+        )
+        if result.is_err or not isinstance(result.return_value, SystemConfig):
+            raise StartupFailure("Unable to determine system configuration of worker")
+
+        # NOTE: Increase the specifier set here if there is a breaking change to this
+        if Version(result.return_value.sdk_version) not in SpecifierSet(">=0.4.1"):
+            # TODO: set to next breaking change release before release
```

Comment:
Noting this needs to be updated to 0.5.0 before 0.5.0 actually releases (temporarily breaking it, but shouldn't need any further updates at least until v1.0)